File: shopping_2.0/parent/shopping_web_manager/src/main/webapp/js/service/brandService.js
//定义service
/*定义控制器*/
app.service('brandService',function ($http) {

    //查询所有
    this.findAll=function () {
      return  $http.get('/brand/findAll.do');
    };

    /*条件查询*/
    this.search = function (pageNum,pageSize ,searchEntity) {
        return $http.post('/brand/search.do?pageNum='+pageNum+"&pageSize="+pageSize,searchEntity);
    };
    /*添加品牌信息*/
    this.add =function (entity) {
        return   $http.post( '/brand/add.do',entity);
    };

    /*更新品牌信息*/
    this.update =function (entity) {

        return  $http.post('/brand/update.do',entity);

    };

    /*根据id查询商品信息 进行回显*/
    this.findOne=function (id) {
        return     $http.get('/brand/findOne.do?id='+id);
    };


    //删除品牌信息
    this.del =function (Ids) {
        return  $http.get('/brand/delete.do?ids='+Ids);
    };


});

File: shopping_2.0/parent/shopping_interface/src/main/java/com/its/core/service/brand/BrandService.java
package com.its.core.service.brand;

import com.its.core.entity.PageResult;
import com.its.core.pojo.good.Brand;

import java.util.List;

public interface BrandService {

    /**查询全部品牌*/

    public List<Brand> findAll();

    /*查询商品分页显示*/
    public PageResult findPage(Integer pageNum, Integer pageSize);

    /*列表查询*/
    public PageResult search(Integer pageNum, Integer pageSize,Brand brand);
    /*添加商品*/
    public void add (Brand brand);
    /*根据id 查寻 进行数据回显*/
    public Brand findOne(Long id);
    /*更新品牌信息*/
    public void update(Brand brand);
    /*删除品牌名称*/
    public  void  delete(Long[] ids);

}
